### PR TITLE
fix(VBadge): fix delay on closing badge if transition is disabled

### LINKED ISSUE #7282: [Bug Report] delay on closing badge if transition is disabled
### Versions and Environment
**Vuetify:** 2.0.0-alpha.20
**Vue:** 2.6.10
**Browsers:** Chrome 74.0.3729.157
**OS:** Windows 10

### Steps to reproduce
Click the button twice

### Expected Behavior
Badge shows instantly on the first click but hides with delay on the second click

### Actual Behavior
Badge should hide instantly

### Reproduction Link
<a href="https://codepen.io/anon/pen/rgdxrX" target="_blank">https://codepen.io/anon/pen/rgdxrX</a>

### Other comments
Bug exists in 1.5.x as well, there's no prop for controlling open/close delay (even with such prop there should be no delay)

### PR DESCRIPTION
**Pull request by beatgrabe**
## Description

If the transition property of a `VBadge` component is unset like:

```
<v-badge transition="" :value="show"></v-badge>
```

it should prevent to badge from actually making a transition when `value` changes.

## Motivation and Context

Fixes #7282 

## How Has This Been Tested?

unit | visually

## Markup:

<details>

```vue
// Paste your FULL Playground.vue here
```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library) 

Fixes #7282

### CODE REVIEW

**Comment by beatgrabe** [comment]
What about fixing in `1.5.x`?

**Comment by jacekkarczmarczyk** [comment]
We might cherry pick it once we create a branch for 1.5 LTS, although not sure if we have already specified workflow for this

File: packages/vuetify/package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "build:dist": "rimraf dist && cross-env NODE_ENV=production webpack --config build/config.js --progress --hide-modules",
     "tsc": "tsc",
     "build:lib": "node build/build-lib.js",
-    "debug:test": "cross-env NODE_ENV=test node --inspect --inspect-brk ./node_modules/jest/bin/jest.js --no-cache -i --verbose",
+    "debug:test": "cross-env NODE_ENV=test node --inspect --inspect-brk ../../node_modules/jest/bin/jest.js --no-cache -i --verbose",
```
Comment (johnleider) [inline packages/vuetify/package.json]: Does this have anything to do with the PR?

Comment (beatgrabe) [reply packages/vuetify/package.json]: I debugged my test with…

Comment (jacekkarczmarczyk) [reply packages/vuetify/package.json]: That shouldn't be in this PR, either there is a problem with running our tests which we are not aware of and should be fixed in a separate issue or there is not problem and this change should be removed from PR

Comment (beatgrabe) [reply packages/vuetify/package.json]: → https://github.com/vuetifyjs/vuetify/pull/7351

**Pull request merged**